index/scorch/scorch.go

```diff
@@ -526,7 +526,7 @@ func (s *Scorch) Analyze(d *document.Document) *index.AnalysisResult {
 			rv.Analyzed[i] = tokenFreqs
 			rv.Length[i] = fieldLength
 
-			if len(d.CompositeFields) > 0 {
+			if len(d.CompositeFields) > 0 && field.Name() != "_id" {
```

Comment:
great catch -- looks good

(separately, another day... looks like "_id" ought to be refactored into a global var or const somewhere for scorch, if not already)

Reply:
Sure @steveyen , I shall make the "_id" const in a separate PR soon.

thanks @mschoch for bringing that point,
Let me create a new ticket to track that down?